cocotb/binary.py

```diff
@@ -279,6 +279,11 @@ def get_value_signed(self):
     def set_value(self, integer):
         self._str = self._convert_to[self.binaryRepresentation](integer)
 
+    @property
+    def is_resolvable(self):
+        """Does the value contain any X's?  Inquiring minds want to know"""
+        return any(char in self._str for char in BinaryValue._resolve_to_error)
```

Comment:
```suggestion
        return not any(char in self._str for char in BinaryValue._resolve_to_error)
```
Fix for previously incorrect change to one-liner